manual_readme_content.md

```diff
@@ -0,0 +1,25 @@
+[comment]: # " File: README.md"
```

Comment:
I think we can remove this whole file a few apps I've checked don't have this license info here